### DISCUSSION TITLE
node-typescript template errors with es build compiling when using express and serverless-http

### DISCUSSION THREAD

**Discussion by andresalbab** [serverless/serverless#9973] [Q&A]
I used the node-typescript template to create an express handler. Previously, web pack was being used to compile and generate the build. I was able to successfully deploy to AWS but few days ago I was having some errors with webpack. I checked the repo and 2 days ago there was a new commit switching from webpack to esbuild.

I made the changes accordingly on my project and now I am having this error when I run it with `serverless offline start` or `Serverless deploy`:
```
src/functions/main/handler.ts:16:12: warning: Calling "express" will crash at run-time because it's an import namespace object, not a function (make sure to enable TypeScript's "esModuleInterop" setting)
    16 │ const app = express()
       ╵             ~~~~~~~
   src/functions/main/handler.ts:4:7: note: Consider changing "express" to a default import instead
     4 │ import * as express from 'express'
       │        ~~~~~~~~~~~~
       ╵        express
```
I have set "esModuleInterop" property to true in my `tsconfig.json` and I am still having this error.

Any clues or help? Thanks

**Comment by mnapoli** [comment]
Just to be sure, let's ping @fredericbarthelet in case there's any regression in the TypeScript build?

If not, sorry for the ping 😅
